integrations/basic-agent-memory.md

```diff
@@ -51,7 +51,7 @@ memory_tool = Tool(name="Memory",
                    pipeline_or_node=memory_node,
                    description="Your memory. Always access this tool first to remember what you have learned.")
 
-prompt_node = PromptNode(model_name_or_path="text-davinci-003", 
```

Comment:
recommended replacement is `gpt-3.5-turbo-instruct` https://platform.openai.com/docs/deprecations/instructgpt-models

Reply:
do you think this should be the replacement for all text-davinci-003 models or just in this integration? And if yes, why?

Reply:
For all of them. It's what OpenAI recommends. They are also saying that `gpt-3.5-turbo-instruct` was trained in a similar way to `text-davinci-003`. I would assume that `gpt-3.5-turbo-instruct` is better at following instructions than `gpt-3.5-turbo`.